### DISCUSSION TITLE
Remove from playlist by range incorrect

### DISCUSSION THREAD

**Discussion by klassikmann** [rern/rAudio#1778] [Fixed Issues]
There is a new option in "Remove From Playlist" to define a range of playlist entries:
 ![grafik](https://github.com/user-attachments/assets/ea1491a9-6989-4f99-87ee-53cde12a96b9)
I have the impression that the entries actually removed are at random compared to the range defined.

**Comment by klassikmann** [comment]
An example:

That's the playlist:
<img src="https://github.com/user-attachments/assets/4709da31-d997-476e-b00d-0f3021437b25" width=450>

Than I use the "Range ..." option in "Remove From Playlist" to remove the entries 2 til 4:
<img src="https://github.com/user-attachments/assets/76d73334-8207-467b-9b91-af8b77cefcd7" width=450>

What I get is this:
<img src="https://github.com/user-attachments/assets/45642fd5-6014-4ae6-ac09-de4cba309a22" width=450>

**Reply by rern** [reply]
Please try the UPDATE.

**Reply by klassikmann** [reply]
@rern 
Seems to be fixed

btw.: Some error messages during UPDATE, probably not related to the issue:
```
grep: /srv/http/data/system/relays.conf: No such file or directory
grep: /srv/http/data/system/relays.conf: No such file or directory
sed: can't read /srv/http/data/system/relays.conf: No such file or Directory
```
